<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUI</class>
 <widget class="QWidget" name="MainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Annotation Tool</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>691</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bar">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>510</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="input">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QPushButton" name="folder_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="video_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Video</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="xmlgroup">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>130</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>XML</string>
   </property>
   <widget class="QPushButton" name="loadXML">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load XML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveXML">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save XML</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="eventgroup">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>240</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Event</string>
   </property>
   <widget class="QPushButton" name="startEvent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Event</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endEvent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End Event</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelEvent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel Event</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="controls" native="true">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>450</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="nextSpeed">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevSpeed">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="arrow">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>494</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/arrow.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="event">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
